backend/nginx.conf

```diff
@@ -8,31 +8,31 @@ http {
   server {
```

Comment:
I think we should discard this file from the PR, unless there is a specific need for it in master.